CICD/ExtensionMethods.cs

```diff
@@ -889,6 +889,27 @@ public static async Task<Milestone> CloseMilestone(this IMilestonesClient client
         return updatedMilestone;
     }
 
+    /// <summary>
+    /// Returns a value indicating whether or not the GitHub issue is a standard issue.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <returns><c>true</c> if a standard issue.</returns>
+    public static bool IsIssue(this Issue issue) => issue.PullRequest is null;
+
+    /// <summary>
+    /// Returns a value indicating whether or not the GitHub issue is a pull request.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <returns><c>true</c> if a standard issue.</returns>
+    public static bool IsPullRequest(this Issue issue) => issue.PullRequest is not null;
+
+    /// <summary>
+    /// Returns a value indicating if a GitHub issue is a release to do issue.
```

Comment:
Refactor code comments

```suggestion
    /// Returns a value indicating whether or not a GitHub issue is a release to do issue.
```